hedera-node/hedera-smart-contract-service-impl/src/test/java/com/hedera/node/app/service/contract/impl/test/handlers/ContractCallHandlerTest.java

```diff
@@ -46,6 +50,8 @@
 
 @ExtendWith(MockitoExtension.class)
 class ContractCallHandlerTest extends ContractHandlerTestBase {
+    public static final long INTRINSIC_GAS = 21064L;
```

Comment:
Maybe change the name to something more descriptive, like `INTRINSIC_GAS_FOR_0_ARG_METHOD`?